qlty-check/src/executor.rs

```diff
@@ -59,19 +59,18 @@ impl Executor {
     }
 
     pub fn install_and_invoke(&self) -> Result<Results> {
-        let install_result = self.install()?;
+        let install_messages = self.install()?;
         self.run_prepare_scripts()?;
         let mut result = self.invoke()?;
 
-        for message in install_result {
-            result.messages.push(message);
-        }
+        result.messages.extend(install_messages);
 
         Ok(result)
     }
 
     pub fn install(&self) -> Result<Vec<Message>> {
         let mut install_messages = vec![];
+        // let mut jobs = vec![];
```

Comment:
Can you please remove the commented code in this one? (A couple more locations below)